.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ on:
 
 jobs:
   unit_tests_rb:
-    name: "Unit tests | Ruby"
+    name: 'Unit tests | Ruby'
```

Comment:
Can you get your editor to stop reformatting every YAML file it touches? :laughing:

Reply:
😅